template css adjustments

RAVMN opened the pull request:
Changes in the css to:
*Delete searchbox text rotation (please check if the other things work, my screen is non-retina and didn't see other changes
*Adress text overflow in privacity checkbox in group's creation page

/cc @hummingbird-me/staff

RAVMN committed:
Delete rotation

It causes the text inside the search box in header to be escalonated

RAVMN committed:
Adjusting privacy selector

Set min-height because if the text is larger it overflows (for example look the spanish translation)

wopian commented on app/styles/pages/_groups.scss:
```diff
@@ -701,7 +701,7 @@ $primary: #00cc99;
       margin: 5px;
       padding: 0;
-      height: 140px;
+      min-height: 160px;
```
.radio-card-group .card-desc

Margin-bottom: 0

---

.radio-card-group .radio-card label

Remove height/min height

RAVMN committed:
Update _groups.scss

140px also works

RAVMN committed:
Update _library.scss

Adjusted width for spanish (the button says Editar, and with the R it gets divided into two lines). Maybe it should get changed to 85.

RAVMN commented:
The file "_library.scss" maybe should be further modified. With min-width the progress column gets resized when a chapter has more caps, but only the box for that anime. With 115px two digit episodes anime are ok, and with 130 I get Detective Conan ok (974 ep).. I've no idea what happens withh 1000+ ep :P
